server/src/main/scala/com/cloudera/livy/server/LivyServer.scala

```diff
@@ -142,6 +144,20 @@ class LivyServer extends Logging {
       }
     }
 
+    // Servlet for hosting static files such as html, css, and js
+    // Necessary since Jetty cannot set it's resource base inside a jar
+    val staticResourceServlet = new ScalatraServlet {
+      get("/*") {
```

Comment:
I wanted to separate the static file hosting servlet from the ui servlet, this was just a preference on my part. I felt that `UIServlet` calls should only return html for consistency, whereas the static resources are files